sdk/identity/test-resources-pre.ps1

```diff
@@ -9,10 +9,42 @@ param (
     $RemainingArguments,
 
     [Parameter()]
-    [switch] $CI = ($null -ne $env:SYSTEM_TEAMPROJECTID)
+    [switch] $CI = ($null -ne $env:SYSTEM_TEAMPROJECTID),
+
+    [Parameter()]
+    [hashtable] $AdditionalParameters = @{}
     
 )
 
+if (!$CI) {
```

Comment:
I think this file is also missing some of the work from recent commits and should be added back https://raw.githubusercontent.com/Azure/azure-sdk-for-js/main/sdk/identity/test-resources-pre.ps1  👍